src/Microsoft.VisualStudio.Web.CodeGeneration.Design/Microsoft.VisualStudio.Web.CodeGeneration.Design.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Code Generation tool for ASP.NET Core. Contains the dotnet-aspnet-codegenerator command used for generating controllers and views. </Description>
     <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
-    <RuntimeFrameworkVersion>1.1.2</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>1.1.3</RuntimeFrameworkVersion>
```

Comment:
Nor this.

Reply:
Will we have it for the patch release though? 

This is the inside man exe, and will be in the project's dependency graph.